### PR TITLE
Restyle Validator detail page

### PR DESCRIPTION
**Pull request by mr-pavlenko**
- Restyle Validator Detail page

screencapture-localhost-8080-validators-odinvaloper1cgfdwtrqfdrzh4z8rkcyx8g4jv22v8wgav3rjx-2022-09-05-12_56_10
screencapture-localhost-8080-validators-odinvaloper1cgfdwtrqfdrzh4z8rkcyx8g4jv22v8wgav3rjx-2022-09-05-12_56_34

### CODE REVIEW

File: src/components/Tag.vue [hunk 1]
```diff
@@ -1,21 +1,15 @@
...
-</script>
-
 <script lang="ts" setup>
 import { computed } from 'vue'
 
-const props = withDefaults(defineProps<{ type: TagType }>(), {
-  type: TagType.Success,
+const props = withDefaults(defineProps<{ type: string; text: string }>(), {
+  type: 'success',
+  text: 'Success',
```
Comment (nikita1tup) [inline src/components/Tag.vue:12]: use const

File: src/components/ValidatorInfo.vue [hunk 2]
```diff
@@ -61,14 +61,13 @@
...
-      <div class="validator-info__description card-frame">
-        <div class="validator-info__description-title">Information</div>
-        <div class="validator-info__description-item">
-          <span class="validator-info__description-item-title"
-            >Delegator shares</span
-          >
-          <span class="validator-info__description-item-value">
+    <div class="app-table">
+      <div class="app-table__row validator-info__line">
+        <div class="app-table__cell app-table__cell-label">
```
Comment (nikita1tup) [inline src/components/ValidatorInfo.vue:66]: i think need app-table__cell app-table__cell--label

**Review by nikita1tup — CHANGES_REQUESTED** [review]

File: src/components/Tag.vue (hunk 1 above)
Comment (mr-pavlenko) [reply src/components/Tag.vue]: Actually, don't think that defaults are really needed here, so removed them at all

File: src/components/ValidatorInfo.vue (hunk 2 above)
Comment (mr-pavlenko) [reply src/components/ValidatorInfo.vue]: Done

**Pull request merged**